examples/python/output_ui/app.py

```diff
@@ -1,36 +1,30 @@
 from shiny import render
-from shiny.express import ui, input
+from shiny.express import ui, input, output, expressify
```

Comment:
Import `render` from `shiny.express`. We should change this now that we noticied but we'll also need to do a pass before merging any `rc-express` branches.